test/unit/utils/component-mappings.test.js

```diff
@@ -0,0 +1,40 @@
+import test from "ava";
+import { getSanitizedComponentMapping } from "../../../src/utils/component-mappings";
+
+test("getSanitizedComponentMapping", t => {
```

Comment:
Nice that we have the groundwork for writings some tests now. There are definitely a number of things we have that should be very easily testable.